src/app/public/src/modules/sidebar/sidebar-wrapper.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="stache-sidebar-wrapper" [ngClass]="{'sidebar-closed': sidebarClosed, 'sidebar-open': !sidebarClosed  }">
+  <div class="stache-sidebar-header">
+    <button *ngIf="!sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="closeSidebar()" aria-label="sidebar close">
+      <i class="fa fa-chevron-left" aria-hidden="Close"></i>
```

Comment:
Curious why you set `aria-hidden` to "Close"?

Reply:
Honestly I didn't know what the aria tags did, just copy pasta'd going back now to make sure I understand them,